sqs/deletequeue.js

```diff
@@ -0,0 +1,17 @@
+const AWS = require('aws-sdk');
+
+AWS.config.loadFromPath('./config.json');
+
+const sqs = new AWS.SQS({ apiVersion: '2012-11-05' });
+
+const params = {
+  QueueUrl: 'https://sqs.us-west-2.amazonaws.com/504566468641/test.fifo',
```

Comment:
If you are using this to delete the queue, you might want to list the queues and remove whichever one needs deleting. Or maybe drive this from some sort of variable instead.